compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TypeChecker.java

```diff
@@ -539,6 +539,9 @@ public void visit(BLangTableLiteral tableLiteral) {
         if (tableConstraint.tag == TypeTags.NONE) {
```

Comment:
Not related to this PR, but I think we should be able to remove this now since a constraint is mandated by the grammar itself.

Reply:
+1